docs/best-practices.md

```diff
@@ -3,32 +3,8 @@ Best Practices
 
 This page is a placeholder for best practices we recommend you to review while implementing Vue InstantSearch.
 
-
-## Custom Component
```

Comment:
I think if I am writing a custom component I want to see/find the best practices on this page rather than on a dedicated page, mixed with many other things.

If we had maybe a lot of different best practices on many subjects yes why not. But even that I think I would prefer not having to go through a generic "best practices" webpage to get the information I want.